contributing/map.rst.inc

```diff
@@ -33,3 +33,7 @@
   * :doc:`Respectful Review comments </contributing/community/review-comments>`
   * :doc:`Community Reviews </contributing/community/reviews>`
   * :doc:`Other Resources </contributing/community/other>`
+
+* **Diversity**
```

Comment:
Should this be just *Diversity* or *Diversity Initiative*?

Reply:
Diversity sounds good to me.